entity-types/infra-kubernetes_persistentvolume/definition.yml

```diff
@@ -8,78 +8,77 @@ dashboardTemplates:
-    #- compositeIdentifier:
-    #    separator: ":"
-    #    attributes:
-    #      - k8s.cluster.name
-    #      - persistentvolume
-    #  encodeIdentifierInGUID: true
-    #  name: persistentvolume
-    #  conditions:
+    # kube-state-metrics data via opentelemetry prometheusReceiver
+    - compositeIdentifier:
+        separator: ":"
+        attributes:
+          - k8s.cluster.name
+          - persistentvolume
+      encodeIdentifierInGUID: true
+      name: persistentvolume
+      conditions:
         # kube-state-metrics persistentvolume prefix
-    #    - attribute: metricName
-    #      prefix: kube_persistentvolume_
+        - attribute: metricName
+          prefix: kube_persistentvolume_
         # identifier attributes
-    #    - attribute: persistentvolume
-    #      present: true
-    #    - attribute: k8s.cluster.name
-    #      present: true
+        - attribute: persistentvolume
+          present: true
+        - attribute: k8s.cluster.name
+          present: true
         # open telemetry
-    #    - attribute: newrelic.source
-    #      value: 'api.metrics.otlp'
+        - attribute: newrelic.source
+          value: "api.metrics.otlp"
         # if service.name is present, it's a service
-    #    - attribute: service.name
-    #      present: false
+        - attribute: service.name
+          present: false
         # value added for test entities only
-    #    - attribute: newrelicOnly
-    #      value: "true"
-    #  tags:
-    #    persistentvolume:
-    #      entityTagNames: [k8s.volumeName]
-    #    k8s.cluster.name:
-    #      entityTagNames: [k8s.clusterName]
-    #    phase:
-    #      multiValue: false
-    #      entityTagNames: [k8s.persistentvolume.statusPhase]
-    #    claim_namespace:
-    #      entityTagNames: [k8s.pvcNamespace]
-    #    name:
-    #      entityTagNames: [k8s.pvcName]
-    #    storageclass:
-    #      entityTagNames: [k8s.persistentvolume.storageClass]
-    # Infrastructure event data via opentelemetry 
-    #- compositeIdentifier:
-    #    separator: ":"
-    #    attributes:
-    #      - k8s.cluster.name
-    #      - k8s.object.name
-    #  encodeIdentifierInGUID: true
-    #  name: k8s.object.name
-    #  conditions:
-    #    - attribute: event.name
-    #      value: InfrastructureEvent
-    #    - attribute: k8s.object.kind
-    #      value: PersistentVolume
+        - attribute: newrelicOnly
+          value: "true"
+      tags:
+        persistentvolume:
+          entityTagNames: [k8s.volumeName]
+        k8s.cluster.name:
+          entityTagNames: [k8s.clusterName]
+        phase:
+          multiValue: false
+          entityTagNames: [k8s.persistentvolume.statusPhase]
+        claim_namespace:
+          entityTagNames: [k8s.pvcNamespace]
+        name:
+          entityTagNames: [k8s.pvcName]
+        storageclass:
+          entityTagNames: [k8s.persistentvolume.storageClass]
+    # Infrastructure event data via opentelemetry
```

Comment:
Do we need to enable entity synthesis from events if the same entity is supposed to be created by the K8s OTel metrics?